### ISSUE TITLE
Setup drone.io as a travis backup

### ISSUE THREAD

**Issue by smarterclayton** [GoogleCloudPlatform/kubernetes#718]
https://drone.io/github.com/smarterclayton/kubernetes/4 is an example, we don't need the build hooks or the email but it would be nice to have a fallback test for pulls.

Configuration in that is 

```
hack/build-go.sh
hack/test-go.sh
go get github.com/coreos/etcd
PATH=$HOME/gopath/bin:$PATH hack/test-cmd.sh
PATH=$HOME/gopath/bin:$PATH hack/integration-test.sh
```

I'll create a .drone.yml for this as well.

**Comment by ixdy** [comment]
We already have Shippable running builds in parallel with Travis. Do we need a third CI?

**Comment by smarterclayton** [comment]
No.

**Comment by zmerlynn** [comment]
They could all vote on each PR!

In all seriousness, the only point would be if one were configured to do
fast checks. At the meetup, I made an off-hand suggestion (based on
@cjcullen's joke) that we could have one CI that's just configured to do
the fast, totally reliable checks, like gofmt.

But it would need to be clear. Right now the display is totally muddy and
even our on-call people, who should know better, end up waiting for
Shippable when it's a waste of time.

**Comment by smarterclayton** [comment]
Do we have issues for the Shippable flakes?

**Comment by zmerlynn** [comment]
I don't know of one. And some of them are weird. I can't decide if we
should keep it on, except for that whole day Travis was down for
maintenance.

@brendanburns was the champion there, though.
